apps/backend/backend-main/src/modules/user/user.controller.ts

```diff
@@ -112,8 +112,8 @@ export class UserController {
    * GET /user/list
    */
   @Get("list")
-  @UseGuards(AuthGuard)
-  @Roles(UserRoleEnum.ADMIN)
+  // @UseGuards(AuthGuard)
+  // @Roles(UserRoleEnum.ADMIN)
```

Comment:
Shouldn't this be there?
```suggestion
  @UseGuards(AuthGuard)
  @Roles(UserRoleEnum.ADMIN)
```